Source/SwiftLintFramework/Extensions/Array+SwiftLint.swift

```diff
@@ -24,7 +24,7 @@ extension Array where Element: Equatable {
     }
 }
 
-extension Array {
+public extension Array {
```

Comment:
I'm not thrilled about exposing these utilities as part of SwiftLintFramework's public API. Can we instead create a simlink to this file in `Source/swiftlint/...` so we can use it from both targets without polluting our public API?

Reply:
I didn't think of symlinking here—sounds like a much better alternative.